controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/configserver/NodeRepository.java

```diff
@@ -122,7 +123,9 @@ private static Node toNode(NodeRepositoryNode node) {
                         clusterTypeOf(node.getMembership()),
                         node.getWantToRetire(),
                         node.getWantToDeprovision(),
-                        Optional.ofNullable(node.getReservedTo()).map(TenantName::from));
+                        Optional.ofNullable(node.getReservedTo()).map(TenantName::from),
+                        DockerImage.fromString(node.getWantedDockerImage()),
```

Comment:
Current & wanted docker image fields are nullable, this needs to be handled similar to `versionFrom()`

Reply:
Yes, thanks, fixed